jenkins.go

```diff
@@ -503,7 +512,7 @@ func CreateJenkins(base string, auth ...interface{}) *Jenkins {
 		base = base[:len(base)-1]
 	}
 	j.Server = base
-	j.Requester = &Requester{Base: base, SslVerify: false}
+	j.Requester = &Requester{Base: base, SslVerify: true}
```

Comment:
I understand your design, thanks. But I think it not convenient to get and set caCert in this way. Why not let user add the cert to host's root CA set, which will be read in default.